kani-compiler/src/codegen_cprover_gotoc/codegen/function.rs

```diff
@@ -53,7 +53,7 @@ impl<'tcx> GotocCtx<'tcx> {
                 var_type,
                 self.codegen_span(&ldata.source_info.span),
             )
-            .with_is_hidden(!ldata.is_user_variable())
+            .with_is_hidden(!self.is_user_variable(&lc))
```

Comment:
Could this change be causing your issues somehow? Since you're getting this information from debug information now, it may be different from what you had before, so some logic continues to try to extract attributes from closures when it wasn't before? Or did it actually work before to extract attributes from closures?

Reply:
I don't think this is related, since this code runs after the code that check for attributes.



> Or did it actually work before to extract attributes from closures?

Yes, this code actually used to run for closures before without crashing. I just double checked our main branch. That said, it always returns an empty list, so I don't think we are missing much. It was just cleaner before.